getting-started.adoc

```diff
@@ -1,154 +1,192 @@
 [[getting-started-with-raincatcher]]
```

Comment:
I personally found the table of contents that were in these docs helpful. Is there a reason they were removed? Is it because whatever AsciiDoc rendering solution we would use does that automatically?